proto/cosmos/crypto/multisig/v1beta1/multisig.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package cosmos.crypto.multisig.v1beta1;
```

Comment:
Should we remove `v1beta1` here? I believe in `cosmos.crypto.*` we kind of decided not to put any v1beta1 anywhere

Reply:
It made sense to remove the versioning for `PubKey` types which are then used to generate addresses but it might be better to keep it for `MultiSignature` and `CompactBitArray` like it's the case for the rest of the proto types.